packages/xstate-store/src/store.ts

```diff
@@ -341,3 +341,8 @@ export function createStoreTransition<
     return { ...snapshot, context: currentContext };
   };
 }
+
+// create a unique 6-char id
+export function uniqueId() {
+  return Math.random().toString(36).substring(2, 8);
```

Comment:
this would have the same effect but it would be shorter :D
```suggestion
  return Math.random().toString(36).slice(6);
```